vocab_files/attribute_concepts/survey-design-and-justification.ttl

```diff
@@ -9,7 +9,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     a skos:Concept ;
     dcterms:source "McCallum K, Bignall J, Laws M, O’Neill S, Sparrow B. (unpublished draft) ‘Targeted Surveys module’ in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "A free text summary of the targeted survey design and justification for conducting the same." ;
+    skos:definition "The survey deign and justification includes any details on the field survey methods being implemented (i.e. combination of methods used), what the survey design is based on (e.g. EPBC survey guidelines, scientific paper) and what is being targeted." ;
```

Comment:
"The survey design and justification include any details on the field survey methods being implemented (i.e. combination of methods used), what the survey design is based on (e.g. EPBC survey guidelines, scientific paper) and what is being targeted."

Reply:
I have fixed few typos